### ISSUE TITLE
NO PUEDO COMPILAR

### ISSUE THREAD

**Issue by jose350** [Desaroll/LoL-Ping-Indicator#1]
Hola, lo que pasa es que soy un aficionado, me parecio genial esta herramienta, me gustaria modificar las direcciones IPs para usarlas en PUGB ya que este no tiene como mostrar el ping en la partida y me seria de utilidad, las IPs las modifico sin problemas, pero como no tengo conocimientos de desarrollo cuando voy a ir a compilar me manda errores y nunca puedo compilar, alguna solucion que me puedas ayudaro decirme como hacerlo, mil gracias

**Comment by Desaroll** [comment]
Personalmente, esto lo armé con Code::Blocks, así que mucho estaba automatizado, pero si me dices que errores te tira al compilar, los vamos solucionando :-)

El requisito principal debería ser SFML.

A todo esto, el programa usa un método de medir el ping algo... Complicado, por que los servidores de LoL (Al menos los de LAS) no responden a los comando `ping`. ¿Sabes si es el caso para los de PUBG? El código se simplifica mucho si responden al comando.

**Comment by jose350** [comment]
Los de PUGB son libres, se dejan medir, incluso en la red estan la lista de todos los servidores globales y funciona sin problemas

**Comment by jose350** [comment]
Me gustaria ayudarte para que tu programa funcione con este y mas juegos, sabes es un programa con mucho potencial, para juegos que necesite un saber el ping dentro de la partida, te comparto un enlace de la pagina donde estan todas las IPS de PUGB para que cheques si sirve, de verdad que te agrasdesco tu ayuda

**Comment by jose350** [comment]
https://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region
este es el listado oficial y esta es la pagina de donde hago el ping por la web
https://pubgserverping.com/

**Comment by Desaroll** [comment]
Yep, gran parte del código "complicado" entonces no es necesario. Toda la parte que hace un "tracert", puedes saltearlo e ir directamente a hacer ping al servidor. No necesitas guardar en config la IP del ultimo nodo ya que solo te interesa el servidor.

No te sugiero partir de mi código y editarlo para hacerlo compatible con PUBG, te quedará mucho código basura. Si quieres que se "vea" igual, copia la parte visual (La función `waitBucle`) y las variables y funciones que usa. Luego tienes que armarte una función que haga el ping directo a las URLs que te indica esa página, y dejar de lado toda la parte del tracert.

Si desde el hilo principal seteas las variables "Number" y "Character", la parte visual sabrá que hacer :3

Estoy ocupado con otros proyectos y ni siquiera juego al PUBG, así que no puedo hacerlo por ti, pero puedo ayudarte si lo intentas y te quedas trabado en alguna parte.

Suerte!

**Comment by jose350** [comment]
Mil gracias lo haré
